src/main/routes/receiver.ts

```diff
@@ -157,11 +158,18 @@ export default express.Router()
           // re-set state cookie as it was cleared above, we need it in this case
           cookies.set(stateCookieName, req.query.state, { sameSite: 'lax' })
           return res.redirect(FirstContactPaths.claimSummaryPage.uri)
+        } else {
+          const letterHolderId: string = getLetterHolderId(req, user)
```

Comment:
Can there be multiple unlinked cases?

Reply:
yes, was just about to point this out, the `getLetterHolderId` function cheats because it knows that on registration there will only ever be 1.

Reply:
You're right. We should cover it as well, even if it's very unlikely to happen.